docs/en/deployment-guides/terminology.md

```diff
@@ -0,0 +1,25 @@
+---
+slug: /en/architecture/introduction
+sidebar_label: Introduction
+title: Introduction
+sidebar_position: 1
+---
+import ReplicationShardingTerminology from '@site/docs/en/_snippets/_replication-sharding-terminology.md';
+
+These deployment examples are based on the advice provided to ClickHouse users by the ClickHouse Support and Services organization.  These are working examples, and we recommend that you try them and then adjust them to suit your needs.  You may find an example here that fits your requirements exactly. Alternatively, you may have a requirement that your data is replicated three times instead of two, you should be able to add another replica by following the patterns presented here.
```

Comment:
```suggestion
These deployment examples are based on the advice provided to ClickHouse users by the ClickHouse Support and Services organization.  These are working examples, and we recommend that you try them and then adjust them to suit your needs.  You may find an example here that fits your requirements exactly. Alternatively, should you have a requirement where data is replicated three times instead of two, you should be able to add another replica by following the patterns presented here.
```